llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -56903,7 +56903,15 @@ X86TargetLowering::getRegForInlineAsmConstraint(const TargetRegisterInfo *TRI,
       case MVT::v8f16:
         if (!Subtarget.hasFP16())
           break;
-        [[fallthrough]];
+        if (VConstraint)
```

Comment:
Why did you drop the Subtarget.hasVLX()?

Reply:
Good catch. I intended to drop for FP16 because it implies VLX. But I forgot to add back for BF16 which doesn't.